backend/src/main/java/de/neuefische/backend/model/Category.java

```diff
@@ -1,6 +1,6 @@
 package de.neuefische.backend.model;
 
-public record Categories(
+public record Category(
         String categories
```

Comment:
Konsequenterweise solltest Du auch den Namen der Eigenschaft mit umbenennen:  😄  
```suggestion
        String category
```